cmd/s3-gw/app_metrics.go

```diff
@@ -195,25 +195,26 @@ func (m *poolMetricsCollector) register() {
 }
 
 func (m *poolMetricsCollector) updateStatistic() {
-	stat := m.poolStatScraper.Statistic()
+	st := m.poolStatScraper.Statistic()
 
 	m.overallNodeErrors.Reset()
 	m.overallNodeRequests.Reset()
 	m.currentErrors.Reset()
 	m.requestDuration.Reset()
 
-	for _, node := range stat.Nodes() {
+	for _, node := range st.Nodes() {
 		m.overallNodeErrors.WithLabelValues(node.Address()).Set(float64(node.OverallErrors()))
 		m.overallNodeRequests.WithLabelValues(node.Address()).Set(float64(node.Requests()))
 
-		m.currentErrors.WithLabelValues(node.Address()).Set(float64(node.CurrentErrors()))
+		// ???????
+		// m.currentErrors.WithLabelValues(node.Address()).Set(float64(node.CurrentErrors()))
```

Comment:
New SDK stat doesn't have _CurrentErrors_. It is a part of the pool internal logic for each client heal checks. Should we:
1. ignore _currentErrors_
2. fill it with `OverallErrors`
3. Somehow pass these values to statistics?

Reply:
`current` as in "for the current connection"? Monitoring sw can track differences in `overall` counters, but it can't different how many reconnects have happened in-between of those. I'm not sure it has any value though, so I'd probably just ignore it.

Reply:
I think `current` is for the exact node. Ok, I will remove